models/demos/convnet_mnist/convnet_mnist_utils.py

```diff
@@ -0,0 +1,37 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import torch
+import torchvision
```

Comment:
Why are we importing torch and torch vision and not using it?

Reply:
torch cat op and torch vision is used for loading MNIST dataset. As it is pre-processing step, these are implemented in Pytorch